spec/mongoid/interceptable_spec.rb

```diff
@@ -2594,4 +2594,36 @@ class TestClass
       end
     end
   end
+
+  context "when around callbacks for embedded children are enabled" do
+    config_override :around_callbacks_for_embeds, true
+
+    context "when around callback is defined without a yield" do
+      class Mother
+        include Mongoid::Document
+        embeds_many :daughters, cascade_callbacks: true
+      end
+
+      class Daughter
+        include Mongoid::Document
+        embedded_in :mother
+        field :flag_around_save, type: String
+        around_save :log_callback
+
+        private
+
+        def log_callback
+          flag_around_save = true
```

Comment:
Ah, I just noticed the field definition that this probably is intended to reference. Note that without an explicit recipient (e.g. `self`), an assignment will create a local variable.

Also, assuming you were intending to access the field and not create a local variable, you're assigning `true` (boolean) to a `String`-valued field.

Probably what you intended was:

```suggestion
          self.flag_around_save = 'saved'
```

Ultimately, though, I think you can probably get rid of the field definition altogether, and instead just write something to the log here (in keeping with the name of the method, `log_callback`). To do that, you can use the `logger` object that is defined on every Mongoid document:

```suggestion
          logger.debug('callback invoked')
```